src/components/Exercises.js

```diff
@@ -54,32 +55,43 @@ const Exercises = ()=>{
   const navigateExerciseForm = ()=>{
     navigate(`/newExercise`)
   }
-
-  return(
-    <div>
-      <h1>Your Exercises</h1>
-      <button onClick={navigateExerciseForm}>Add New Exercise</button>
-      <ul>
-        {exercisesData.exercises.map(curr=>{
-          let currWorkout 
-          exercisesData.workouts.map((item)=>{
-            if(item._id === curr.workout){
-              currWorkout = item
-            }
-          })
-      
-          return(
-            <li >
-              <h3>Title: {curr.title}</h3>
-              <p>Category: {curr.category}</p>
-              <p>Workout: <button onClick={()=>navigateWorkout(currWorkout._id)}>{currWorkout.title}</button></p>
-              <button onClick={()=>navigateStats(curr.id)}>See Stats</button>       
-            </li> //add functionality to open to exercise page with stats.
-          )
-        })}
-      </ul>
-    </div>
-  )
+  if(exercisesData.exercises.length === 0){
+    return(
+      <div>
+        <h1>Your Exercises</h1>
+        <button onClick={navigateExerciseForm}>Add New Exercise</button>
+        <p>No Exercises Yet</p>
+      </div>
+    )
+  }else{
+    console.log(exercisesData)
+    return(
+      <div>
+        <h1>Your Exercises</h1>
+        <button onClick={navigateExerciseForm}>Add New Exercise</button>
+        <ul>
+          {exercisesData.exercises.map(curr=>{
```

Comment:
We typically use `.map` when we want to create a new array with some kind of modification to the original elements. In both this line and the one a little below it, I would try to use `.forEach()` if possible.